src/script/auth/page/Root.tsx

```diff
@@ -62,6 +58,25 @@ import * as LanguageSelector from '../module/selector/LanguageSelector';
 import {ROUTE} from '../route';
 import {getOAuthQueryString} from '../util/oauthUtil';
 
+/**
+ *  Lazy loading components for routes, which can only be accessed by activated feature flags
+ */
```

Comment:
Lazy loading some components. Using loadable to be able to use webpack code splitting in the future

Reply:
will be removed